tools/tox.ini

```diff
@@ -17,6 +17,7 @@ passenv =
     CUDAARCHS
     CUDACXX
     NVFORTRAN_COMPILER
+    TEST_TEMP_DIR
 deps =
```

Comment:
I think for now there's no need for an additional pytest flag, we can keep using the `--datatest` flag. If we start adding lots of additional tests using serialised data and want to for example run them apart from the rest (because they took long), then I would think about adding a new flag.